### PR TITLE
Update deepFreeze.md

### PR DESCRIPTION
**Pull request by ronib**
fix typo in line 15, and add missing typeof call

### CODE REVIEW

**Comment by davidghf** [comment]
Buying and selling ornamental and energetic stones from different countries without entering Iran, direct delivery from origin to destination

**Comment by davidghf** [comment]
Buying and selling ornamental and energetic stones from different countries without entering Iran, direct delivery from origin to destination

**Comment by davidghf** [comment]
+989016499375
dideban.m@gmail.com

**Comment by davidghf** [comment]
Buying and selling ornamental and energetic stones from different countries without entering Iran, direct delivery from origin to destination

**Comment by davidghf** [comment]
Given the current situation in my country, I want to enter the global market internationally and have a bank account in foreign countries.

**Comment by AmbitionJs** [comment]
unsubscribe




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "30-seconds/30-seconds-of-code"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年7月20日(星期一) 下午3:19
收件人:&nbsp;"30-seconds/30-seconds-of-code"<30-seconds-of-code@noreply.github.com&gt;;
抄送:&nbsp;"Subscribed"<subscribed@noreply.github.com&gt;;
主题:&nbsp;Re: [30-seconds/30-seconds-of-code] Update deepFreeze.md (#1168)





 
Buying and selling ornamental and energetic stones from different countries without entering Iran, direct delivery from origin to destination

**Comment by davidghf** [comment]
Given the current situation in my country, I want to enter the global market internationally and have a bank account in foreign countries.

File: snippets/deepFreeze.md [hunk 1]
```diff
@@ -6,13 +6,13 @@ tags: object,recursion,intermediate
 Deep freezes an object.
 
 Use `Object.keys()` to get all the properties of the passed object, `Array.prototype.forEach()` to iterate over them.
-Call `Object.freeze(obj)` recursively on all properties, checking if each one is frozen using `Object.isFrozen()` and applying `deepFreeze()` as necessary.
+Call `Object.freeze(obj)` recursively on all properties, checking if each object is frozen using `Object.isFrozen()` and applying `deepFreeze()` as necessary.
```
Comment (davidghf) [inline snippets/deepFreeze.md:9]: Given the current situation in my country, I want to enter the global market internationally and have a bank account in foreign countries.

**Comment by AmbitionJs** [comment]
unsubscribe




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "30-seconds/30-seconds-of-code"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年7月20日(星期一) 下午3:20
收件人:&nbsp;"30-seconds/30-seconds-of-code"<30-seconds-of-code@noreply.github.com&gt;;
抄送:&nbsp;"544801019"<544801019@qq.com&gt;;"Comment"<comment@noreply.github.com&gt;;
主题:&nbsp;Re: [30-seconds/30-seconds-of-code] Update deepFreeze.md (#1168)





 
Given the current situation in my country, I want to enter the global market internationally and have a bank account in foreign countries.

**Comment by davidghf** [comment]
46) را 92. |  8:55 User Profile Edit NO 09147383605 Name and Surname: Davood Ghaffarzadeh National Code: 35935502 Postal Code: 5314866955 Phone: 04153232234 Address: No. 39 Fire Street

**Comment by davidghf** [comment]
46) را 92. |  8:55 User Profile Edit NO 09147383605 Name and Surname: Davood Ghaffarzadeh National Code: 35935502 Postal Code: 5314866955 Phone: 04153232234 Address: No. 39 Fire Street

**Pull request merged**